2022-02-04-qubes-4-1-0.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Qubes OS 4.1.0 has been released!"
+categories: releases
+download_url: /downloads/
+author: Andrew David Wong
+---
+
+At long last, the Qubes 4.1.0 stable release has arrived!
+
+
+## What's new in Qubes 4.1.0?
+
+In case you still haven't heard, Qubes 4.1.0 includes several major new
+features, each of which is explained in depth in its own article.
+
+
+### [Qubes Architecture Next Steps: The GUI Domain]
+
+The GUI domain is a qube separate from dom0 that handles all
+display-related tasks and some system management. This separation allows
+us to more securely isolate dom0 while granting the user more
+flexibility with respect to graphical interfaces.
+
+
+### [Qubes Architecture Next Steps: The New Qrexec Policy System]
+
+Qrexec is is an RPC (remote procedure call) mechanism that allows one
+qube to do something inside another qube. The qrexec *policy* system
+enforces "who can do what and where." Qubes 4.1 brings a new qrexec
+policy format, significant performance improvements, support for socket
+services, and policy notifications that make it easier to detect
+problems.
+
+
+### [New Gentoo templates and maintenance infrastructure]
+
+There are three new flavors of Gentoo templates, as well as an advanced
+infrastructure for automated building and testing, which also supports
+Linux kernel and Arch Linux building and testing.
+
+
+### [Improvements in testing and building: GitLab CI and reproducible builds]
+
+This article explains our work on continuous integration (CI), which
+automates and improves several aspects of the development process, and
+reproducible builds, which improves the security of the build and
+verification process.
+
+
+### [Reproducible builds for Debian: a big step forward]
+
+This article explains the tools and infrastructure we've built to verify
+official package builds by rebuilding them. While this was supposed to
+be possible in theory, making it a reality required significant work,
+including rewriting certain components from scratch.
+
+
+### Improvements, bug fixes, and updated components
+
+In addition to the articles above, there are also  numerous other
+improvements and bug fixes listed in the [release notes] and in the
+[issue tracker].
+
+Finally, Qubes 4.1.0 features the following updated default components:
+
+- Xen 4.14
+- Fedora 32 in dom0
+- Fedora 34 template
+- Debian 11 template
+- Whonix 16 Gateway and Workstation templates
+- Linux kernel 5.10
+
+
+## How to upgrade to Qubes 4.1.0
+
+- If you're already on any 4.1.0 release candidate, simply perform a
+  normal [update].
+- If you're not on a 4.1.0 release candidate, please see [how to upgrade
+  to Qubes 4.1].
```

Comment:
Small suggestion: this part be near the top since it's possible some will just want actionable information.

Reply:
I'm not sure about moving this - downloads are available from the top menu, and upgrade docs are also easily available in the documentation - if someone want to skip the announcement and go straight to install, there is an easy way for it.

That said, there should be also a link to downloads page, for those installing for the first time.

Reply:
> Small suggestion: this part be near the top since it's possible some will just want actionable information.

My sense is that it's pretty common to have "what's new" before "how to install or upgrade," but I dunno. No strong opinion. I would hope that, with clearly-labeled headings, those people are capable of skipping over just one section to get to the part they want.

> That said, there should be also a link to downloads page, for those installing for the first time.

Ah, I forgot. Thanks. Added.